superset/assets/javascripts/dashboard/components/Header.jsx

```diff
@@ -19,7 +19,10 @@ class Header extends React.PureComponent {
     return (
       <div className="title">
         <div className="pull-left">
-          <h1>{dashboard.dashboard_title}</h1>
+          <h1>
+            {dashboard.dashboard_title} &nbsp;
+            <span is class="favstar" class_name="Dashboard" obj_id={dashboard.id} />
```

Comment:
ReactDOM's `is` flag is a way to allow for passing raw unstandard html attributes which in this case I needed if I didn't want to alter the way the "favstar" thing was designed.  Once that flag is on everything is like raw html, so `class` instead of `className`.

We need to redesign FavStar as a react component, in the meantime it's a little hack to make it work as is.